### PR TITLE
Get Dashboard Count API - Get the count of result

### PR DESCRIPTION
**Pull request by YoginiTayade**

### CODE REVIEW

File: src/portal/portal.services.ts [hunk 1]
```diff
@@ -36,180 +38,195 @@ export class PortalService {
           result: null,
         });
       } else {
-        const sb_rc_search = await this.sbrcService.sbrcSearchEL('TeacherV1', {
```
Comment (rushi-tekdi) [inline src/portal/portal.services.ts:39]: @YoginiTayade  remove all commented code from this function

File: src/services/sbrc/sbrc.service.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { HttpService } from '@nestjs/axios';
 import { Injectable } from '@nestjs/common';
 import { AxiosRequestConfig } from 'axios';
+import { log } from 'node:console';
```
[the remark this answers is not in the document]
Comment (YoginiTayade) [reply src/services/sbrc/sbrc.service.ts:4]: Resolved

File: src/services/cred/cred.service.ts [hunk 3]
```diff
@@ -190,4 +188,36 @@ export class CredService {
     }
     return cred_search;
   }
+  //custom
+  //credSearchFilter
+  async credSearchFilter(subjectFilter: any) {
+    let data = JSON.stringify({
+      subject: subjectFilter,
+    });
+    // let data = JSON.stringify({
```
[the remark this answers is not in the document]
Comment (YoginiTayade) [reply src/services/cred/cred.service.ts:197]: Resolved

File: src/portal/portal.services.ts (hunk 1 above)
Comment (YoginiTayade) [reply src/portal/portal.services.ts:39]: Resolved

**Pull request merged**